report/3.Theory/LanguageSpecification.tex

```diff
@@ -76,6 +85,27 @@ \section{Language Specification}
 ["john", "frank"] where {value is "frank"} print
 \end{lstlisting}
 
+\textbf{Grammar for function calls}\\
+There are three different ways to call a function depending on how many parameters must be passed with the function:
+\begin{itemize}
+	\item If no parameters must be passed, then invoking the function identifier is a call in and of itself.
+	\item If one parameter must be passed, then the parameter must appear on the left-hand side of the function identifier.
```

Comment:
"argument" er input - parameter er inde i metoden så der skal stå 

"If one parameter must be passed, then the argument must appear on the left-hand side of the function identifier."